google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -774,6 +774,15 @@ public class ITBigQueryTest {
 
   private static final Set<String> PUBLIC_DATASETS =
       ImmutableSet.of("github_repos", "hacker_news", "noaa_gsod", "samples", "usa_names");
+  private static final Map<String, String> PUBLIC_NON_US_DATASETS =
+      ImmutableMap.<String, String>builder()
+          // Dataset url:
+          // https://console.cloud.google.com/bigquery?project=bigquery-public-data&ws=!1m4!1m3!3m2!1sbigquery-public-data!2sgnomAD_asiane1
+          .put("gnomAD_asiane1", "asia-northeast1")
+          // Dataset url:
+          // https://console.cloud.google.com/bigquery?project=bigquery-public-data&ws=!1m4!1m3!3m2!1sbigquery-public-data!2sgnomAD_eu
+          .put("gnomAD_eu", "EU")
+          .build();
```

Comment:
It'll make the validation earlier if you combine PUBLIC_DATASETS into PUBLIC_NON_US_DATASETS.
I.e.,
private static final Map<String, String> PUBLIC_DATASETS_LOCATION =
...
.put("gnomAD_asiane1", "asia-northeast1")
.put("gnomAD_eu", "EU")
.put("github_repos", "US")
...
.build();

Then for validation:
for (String type : PUBLIC_DATASETS.keySet()) {
assertTrue(datasetNames.contains(type));
assertEquals(PUBLIC_DATASETS_LOCATION.get(type), datasetLocation.get(type));
}

Reply:
Thanks for the suggestion @PhongChuong - addressed it in the latest commit.